cms/static/js/spec/views/login_studio_spec.js

```diff
@@ -0,0 +1,38 @@
+define(['jquery', 'js/factories/login', 'common/js/spec_helpers/ajax_helpers', 'common/js/components/utils/view_utils'],
+function($, LoginFactory, AjaxHelpers, ViewUtils) {
+    'use strict';
+    describe("Studio Login Page", function() {
+        var submit_button;
+
+        beforeEach(function() {
+            loadFixtures('login.underscore');
+            /*jshint unused: false*/
+            var login_factory = new LoginFactory("/home/");
+            submit_button = $('#submit');
+        });
+
+        it('It will disable the submit button once it is clicked', function() {
```

Comment:
Nit: remove the "It " prefix from the test name, because it is intended to be read with the function ``it``,

    it('disables the submit button once it is clicked', function() {